### PR TITLE
Added Debian alternative, fixed root hickup (see issue #414)

### LINKED ISSUE #414: Install and Configure Ubuntu Remote Desktop with XFCE using X2Go
## Problem
Finally on our server, we remove root´s password. The new created user has got sudo rights.
passwd -d root
## Suggested Fix

New user doesn't have sudo rights. After this you can not login to the server is root.

### PR DESCRIPTION
**Pull request by HeartOfGermany**
Fixes #414 

- Removed root password completely, switched to SSH key and linked article
- Added Debian LXDE route for lighter alternative
- Added comparision
- Added optional settings section
- transplanted those changes to the german tutorial

### CODE REVIEW

File: tutorials/install-and-configure-x2goserver/01.de.md [hunk 1]
```diff
@@ -16,19 +16,25 @@ header_img: "header-7"
...
+Es wird eine graphische Benutzeroberfläche installiert. Außerdem wird ein neuer Benutzer angelegt, der über [X2GO](https://wiki.x2go.org/doku.php) verschlüsselt auf seinen Desktop zugreifen kann.
 
 X2Go ist eine kostenlose Terminalserver-Lösung für die Verbindung zu verschiedenen Desktop-Umgebungen wie XFCE, LXDE, Mate usw.
 
 **Voraussetzungen**
 
  - ein neu installierter Server, der in der Hetzner Cloud Console mit dem Basis-Image von ubuntu 20.04 erstellt wurde
  - IP-Adresse
- - Rootpasswort
+ - root Zugang zum Server per SSH Key [See our tutorial for reference](https://community.hetzner.com/tutorials/add-ssh-key-to-your-hetzner-cloud)
```
Comment (rook59) [inline tutorials/install-and-configure-x2goserver/01.de.md:35]: Link text and link to tutorial should be in german

File: tutorials/install-and-configure-x2goserver/01.de.md [hunk 2]
```diff
@@ -38,25 +44,57 @@ Legen Sie eine neue Firewall an. Die Grundeinstellungen in Ihrer Firewall sind i
...
+- xubuntu-desktop > Vollständiger Desktop
+- x2goserver > Server für die x2go Session
+- x2goserver-xsession > Tool für x2go für die Kommunikation zum xServer
+- lxde-core > Minimalistischer Desktop
+- lxrandr > Zwingend notwendig, da sonst keine Auflösungen eingestellt werden können und nur 800x600 funktioniert
+- firefox-esr > Notwendig, da lxde-core keinen Browser beinhaltet
+- xfce4-terminal > Notwendig, da kein Terminal installiert. Benötigt keine Abhängigkeiten, ist daher schlank aber hat die nötigen Funktionen
+- pulseaudio > Notwendig, falls Soundausgabe gefordert ist. Ist dies nicht notwendig, den Eintrag entfernen - spart viel Speicherplatz
+
+Für die Smarten kann man alle bisherigen Befehle in einer einzigen Zeile ausführen:
```
Comment (rook59) [inline tutorials/install-and-configure-x2goserver/01.de.md:82]: I know what you mean but please find an alternative wording for "für die Smarten"

File: tutorials/install-and-configure-x2goserver/01.de.md [hunk 3]
```diff
@@ -38,25 +44,57 @@ Legen Sie eine neue Firewall an. Die Grundeinstellungen in Ihrer Firewall sind i
...
+
+Ubuntu: `apt update && apt upgrade -y && apt install xubuntu-desktop x2goserver x2goserver-xsession -y`
+- Fertig in 16:58 minutes
+- 2x Nutzereingabe notwendig (Auswahl der Standard SSH config und lightdm als Display Manager)
+- Aktuell bug in der repository, der 2 malige Ausführung erfordert
+- Speicherbedarf direkt nach Installation: 5553MB
+- RAM-Verbrauch mit 1 User @ Desktop @ 1080p: 540MB
+
+Debian: `apt update && apt upgrade -y && apt install lxde-core lxrandr x2goserver x2goserver-xsession firefox-esr xfce4-terminal pulseaudio -y`
+- Fertig in 03:17 minutes
```
Comment (rook59) [inline tutorials/install-and-configure-x2goserver/01.de.md:92]: The time completely depends on the server resources, network speed, etc, I would remove it

**Comment by svenja11** [comment]
Closing as there was no activity for a very long time.
The linked issue was fixed with #661

**Pull request closed without merging**